pkg/apis/extensions/v1beta1/types.go

```diff
@@ -286,6 +286,12 @@ type DeploymentStatus struct {
 
 	// Total number of non-terminated pods targeted by this deployment that have the desired template spec.
 	UpdatedReplicas int32 `json:"updatedReplicas,omitempty"`
+
+	// Total number of available pods (ready for at least minReadySeconds) targeted by this deployment.
+	AvailableReplicas int32 `json:"availableReplicas,omitempty"`
```

Comment:
Without pointers I think the field won't appear if the value is 0. Which probably is ok.